apps/ui/src/views/My/Explore.vue

```diff
@@ -1,20 +1,45 @@
+import ProposalIconStatus from '@/components/ProposalIconStatus.vue';
+
 <script setup lang="ts">
+import { explorePageProtocols } from '../../networks';
+import { ProtocolConfig } from '../../networks/types';
+
 const { setTitle } = useTitle();
 const spacesStore = useSpacesStore();
-
-onMounted(() => spacesStore.fetch());
-
 watchEffect(() => setTitle('Explore'));
+onMounted(() => spacesStore.fetch());
+const protocols = Object.values(explorePageProtocols).map((protocol: ProtocolConfig) => ({
+  key: protocol.key,
+  label: protocol.label
+}));
```

Comment:
It feels a bit weird with all those put together without line breaks. Usually we separate different types of composables by whitespace.

Also I feel like protocols should be at the top, not bottom.